modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -121,6 +121,8 @@
 				[@subpath, @path, foo]
 				[, res://test.gd, bar]
 				[/codeblock]
+				[b]Note:[/b] This function is not recursive, i.e. nested objects will not be represented as dictionaries. Properties that have reference types (arrays, dictionaries, objects) will not be duplicated.
```

Comment:
```suggestion
				[b]Note:[/b] This function is not recursive, which means that nested objects will not be represented as dictionaries. Properties passed by reference ([Array], [Dictionary], and [Object] types) will not be duplicated.
```
Additionally/alternatively, and I am somewhat unsure if I am understanding this right:
```suggestion
				[b]Note:[/b] This function is not recursive, which means that nested objects are not be represented as dictionaries. Container and [Object] properties are passed by their original reference and are not be duplicated.
```

Or similar.